types/uppercamelcase/index.d.ts

```diff
@@ -1,8 +1,8 @@
-// Type definitions for uppercamelcase
+// Type definitions for uppercamelcase 3.0
 // Project: https://github.com/samverschueren/uppercamelcase
 // Definitions by: York Yao <https://github.com/plantain-00>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare function upperCamelCase(...args: string[]): string;
 export = upperCamelCase;
-declare namespace upperCamelCase { }
```

Comment:
This is very confusing. Setting the module to `es2015` leads to errors when using the `import upperCamelCase = require('uppercamelcase');` syntax. Which I've expected. What I didn't expect is the output of `esnext` module system which lacks the `require()` statement altogether. What I know is that you can safely substitute `import upperCamelCase = require('uppercamelcase');` with `const upperCamelCase = require('uppercamelcase');`.

But this change has the potential to break a lot of setups. I think the best way to proceed is to revert this change.